projects/aca-content/src/lib/components/search/search-save/dialog/edit/saved-search-edit-dialog.component.ts

```diff
@@ -42,7 +43,11 @@ import { FormControl, FormGroup, Validators } from '@angular/forms';
 })
 export class SavedSearchEditDialogComponent {
   form = new FormGroup({
-    name: new FormControl('', [Validators.required, forbidOnlySpaces]),
+    name: new FormControl('', {
+      validators: [Validators.required, forbidOnlySpaces],
+      asyncValidators: [this.uniqueSearchNameValidator.validate.bind(this.uniqueSearchNameValidator)],
```

Comment:
https://hyland.atlassian.net/browse/ACS-8621
about the name:
"the name  does NOT need to be unique "
has something changed here?

Reply:
I was discussing this with Arabella and we agreed it feels weird for the name not to be unique so I added a change for that, will update the ticket